packages/vuetify/src/util/helpers.ts

```diff
@@ -322,17 +322,15 @@ export const keyCodes = Object.freeze({
   pagedown: 34,
 })
 
-const ICONS_PREFIX = '$vuetify.'
-
-// This remaps internal names like '$vuetify.icons.cancel'
+// This remaps internal names like '$cancel' or '$vuetify.icons.cancel'
 // to the current name or component for that icon.
 export function remapInternalIcon (vm: Vue, iconName: string): VuetifyIcon {
-  if (!iconName.startsWith(ICONS_PREFIX)) {
+  if (!iconName.startsWith('$')) {
     return iconName
   }
 
   // Get the target icon name
-  const iconPath = `$vuetify.icons.values.${iconName.split('.').pop()}`
+  const iconPath = `$vuetify.icons.values.${iconName.split('$').pop()!.split('.').pop()}`
```

Comment:
Deprecate `$vuetify.icons.`?

Reply:
My changes doesn't make it deprecated, both will work.
Not sure if depecrating it is relevant unless everyone upvotes it
`$vuetify.icons.` is more verbose but some people probably will prefer it over just `$`

Reply:
Still with my changes even `$whatever.you.want.name` will resolve to `$vuetify.icons.values.name`